infrastructure/scripts/validate-amplify-permissions.sh

```
@@ -0,0 +1,462 @@
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+CYAN='\033[0;36m'
+NC='\033[0m' # No Color
+
+# Configuration
+ROLE_ARN=${AWS_ROLE_ARN:-""}
+TEST_APP_NAME="macro-ai-frontend-test-validation"
+TEST_STACK_NAME="MacroAiTestStack"
```

Shellcheck (0.10.0)

[warning] TEST_STACK_NAME appears unused. Verify use (or export if used externally).

(SC2034)